back/spec/services/export/xlsx/project_ideas_votes_generator_spec.rb

```diff
@@ -160,7 +160,7 @@
 
       expect(idea_row[header_row.find_index 'ID']).to eq ideas[0].id
       expect(idea_row[header_row.find_index 'Title']).to eq ideas[0].title_multiloc['en']
-      expect(idea_row[header_row.find_index 'Description']).to eq ideas[0].body_multiloc['en']
+      expect(idea_row[header_row.find_index 'Description']).to eq Export::Xlsx::Utils.new.convert_to_text_long_lines(ideas[0].body_multiloc['en'])
```

Comment:
We were already converting HTML content to text for ideation ideas, so this just tests the new alignment with that behaviour.